src/explorer/WebAppProvider.ts

```diff
@@ -3,15 +3,14 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { Location } from 'azure-arm-resource/lib/subscription/models';
 import { WebSiteManagementClient } from 'azure-arm-website';
 import { Site, WebAppCollection } from 'azure-arm-website/lib/models';
-import { workspace, WorkspaceConfiguration } from 'vscode';
-import { createWebApp, SiteClient } from 'vscode-azureappservice';
-import { AzureTreeItem, createAzureClient, createTreeItemsWithErrorHandling, IActionContext, parseError, SubscriptionTreeItem, UserCancelledError } from 'vscode-azureextensionui';
-import { configurationSettings, extensionPrefix } from '../constants';
+import * as appservice from 'vscode-azureappservice';
+import * as extensionui from 'vscode-azureextensionui';
```

Comment:
Why'd you have to do these `appservice ` and `extensionui ` things? I try to avoid it because it makes the code harder to read and the imports are generally handled automatically anyways

Reply:
I think there were two reasons:

1. appservice because I can't have a shadowed variable name `editScmType`.  I'm fine with calling it something else like `configureDeploymentSource` since that's what the UI says anyway

1. extensionui because there were so many things imported that it looked terrible and felt like I might as well just import the whole thing.